### ISSUE TITLE
don't show map if the API does not have latitude and longitude keys

### ISSUE THREAD

**Issue by sayanee** [liccowee/webuildsg-android#2]
E.g. KopiJS 26:

``` json
{
    id: "ehp0085j3mf92096ukm0tks7gs",
    name: "KopiJS #26",
    description: "https://github.com/KopiJS/kopi.js/issues/48",
    location: "Wimbly Lu Chocolates, 15-2 Jalan Riang, Singapore 358987",
    url: "https://github.com/KopiJS/kopi.js/issues/48",
    group_name: "KopiJS",
    group_url: "http://kopi.io/",
    formatted_time: "30 Jul 2016, Sat, 10:00 am",
    start_time: "2016-07-30T02:00:00.000Z",
    end_time: "2016-07-30T05:00:00.000Z",
    platform: "ics"
}
```

This is because sometimes geocoding gives the wrong location and meetup attendees might end up in the wrong area 😅.

E.g. screenshot from the Android app:

![photo 29-7-16 7 28 22 pm](https://cloud.githubusercontent.com/assets/413879/17248335/718dcf22-55cc-11e6-8266-4cbe5786dbf4.jpg)

In iOS I completely remove the map view to avoid any amgibuity:

![photo 29-7-16 8 34 42 pm](https://cloud.githubusercontent.com/assets/413879/17248340/7b9ffc60-55cc-11e6-8c63-c94740702a6d.png)

**Comment by liccowee** [comment]
Fixed. Will upload new build soon.

**Comment by sayanee** [comment]
fixed with 00824f6d38eddfe8277424e4cbc110110f49f60d
